Src/CSharpier/Printer/InvocationExpressionSyntax.cs

```diff
@@ -40,13 +36,12 @@ void Traverse(ExpressionSyntax expression)
                 {
                     Traverse(invocationExpressionSyntax.Expression);
                     printedNodes.Add(
-                        new PrintedNode
-                        {
-                            Doc = this.PrintArgumentListSyntax(
+                        new PrintedNode(
+                            invocationExpressionSyntax,
+                            this.PrintArgumentListSyntax(
                                 invocationExpressionSyntax.ArgumentList
-                            ),
-                            Node = invocationExpressionSyntax
```

Comment:
You're losing information here when switching from initializer to constructor. 
Consider maybe using named parameters like:
```C#
new PrintedNode(
    node: invocationExpressionSyntax,
    doc: this.PrintArgumentListSyntax(
        invocationExpressionSyntax.ArgumentList
    )
)
```

Reply:
yeah that's one thing I didn't like when I was switching these, but I didn't think to name the parameters.